packages/code-highlight/src/code/line-numbers.ts

```diff
@@ -42,12 +42,13 @@ export function codeBlockLinesPlugin() {
           // Adds a line break to the end of each line
           node.children.forEach((child: ElementContent) => {
             if (child.type === 'element' && child.tagName === 'span') {
-              const lastChild: ElementContent =
+              const lastChild: ElementContent | undefined =
```

Comment:
I wish you could use a question mark for this 😢

Reply:
whats the magic word

Reply:
I don't think you can though right? Like that's not a typescript feature (I just wish it was)